conf/distro/rpb-qt5.conf

```diff
@@ -0,0 +1,8 @@
+require conf/distro/include/rpb.inc
+
+DISTRO_NAME = "Reference-Platform-Build-QT5"
+
+PACKAGECONFIG_append_pn-qtbase = " gles2 eglfs fontconfig kms"
```

Comment:
yes it is building on my side as well. should I send an updated patch or will you push your own?